app/models/course.rb

```diff
@@ -1,7 +1,24 @@
-class Course < ApplicationRecord
+# The model representing a course held at HPI
+class Course < SearchableRecord
   validates :name, presence: true
 
   has_and_belongs_to_many :people
   has_many :course_times, dependent: :destroy
   belongs_to :room, optional: true
+
+  def image_or_placeholder
+    "https://via.placeholder.com/512?text=Course"
```

Comment:
There should be a placeholder_course.png in the assets which should be used here